### PR TITLE
[hi] Localize content/en/docs/reference/glossary/cncf.md into Hindi

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/cncf.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+short_description: >
+  क्लाउड नेटिव कंप्यूटिंग फाउंडेशन 
+
+aka:
+tags:
+- community
+---
+ क्लाउड नेटिव कंप्यूटिंग फाउंडेशन (CNCF) स्थायी पारिस्थितिकी तंत्र का निर्माण करता है और
+ आसपास के समुदाय [प्रोजेक्ट्स](https://www.cncf.io/projects/) को बढ़ावा देता है  वह
+ एक माइक्रोसर्विस आर्किटेक्चर के हिस्से के रूप में कंटेनरों को ओर्केस्ट्रेट करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cncf.md:15]:
```suggestion
 क्लाउड नेटिव कंप्यूटिंग फाउंडेशन (CNCF) स्थायी पारिस्थितिकी तंत्र का निर्माण करता है और
 [परियोजना](https://www.cncf.io/projects/) के आसपास समुदाय को बढ़ावा देता है जो 
 एक माइक्रोसर्विस आर्किटेक्चर के हिस्से के रूप में कंटेनरों को ओर्केस्ट्रेट करता है।
```

File: content/hi/docs/reference/glossary/cncf.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+कुबेरनेट्स एक CNCF प्रोजेक्ट है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cncf.md:17]:
```suggestion
कुबेरनेट्स एक CNCF परियोजना है।
```

File: content/hi/docs/reference/glossary/cncf.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<!--more-->
+
+CNCF [Linux फाउंडेशन](https://www.linuxfoundation.org/) का एक उप-आधार है.
+इसका मिशन क्लाउड नेटिव कंप्यूटिंग को सर्वव्यापी बनाना है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cncf.md:22]:
```suggestion
CNCF [Linux फाउंडेशन](https://www.linuxfoundation.org/) का एक उप-संस्थान है।
इसका लक्ष्य क्लाउड नेटिव कंप्यूटिंग को सर्वव्यापी बनाना है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @amolmote

**Review by anubha-v-ardhan — COMMENTED** [review]
Thanks for accepting the change suggestions. Can you squash the commits so we can get this PR ready for merge.

**Comment by jihoon-seo** [comment]
/remove-area blog release-eng
/remove-language de en es fr ja ko pt zh
/remove-sig release

**Pull request closed without merging**